sockeye/inference.py

```diff
@@ -1372,11 +1372,17 @@ def _log_linear_interpolation(predictions):
     def translate(self, trans_inputs: List[TranslatorInput]) -> List[TranslatorOutput]:
         """
         Batch-translates a list of TranslatorInputs, returns a list of TranslatorOutputs.
-        Splits oversized sentences to sentence chunks of size less than max_input_length.
+        Empty or bad inputs are skipped.
+        Splits inputs larger than Translator.max_input_length into chunks of size max_input_length.
+        Translates in batches of Translator.max_batch_size or less if there are fewer inputs/chunks.
+        If there are more inputs/chunks than Translator.max_batch_size, multiple batches of size
+        min(len(remaining_chunks), max_batch_size) are translated in sequence.
```

Comment:
I think the min() expression here confuses more than it helps.

Reply:
Do you have a better phrasing? I tried and failed :)

Reply:
"Splits inputs longer than Translator.max_input_length into segments of size max_input_length, and then groups segments into batches of at most Translator.max_batch_size. Too-long segments that were split are reassembled into a single output after translation."

Reply:
wonderful prose, thank you!

Reply:
I tried to work in a Jane Austen reference, but couldn't pull it off.